File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/DataContext.cs
namespace DDS_Restaurant_Solution.Models
{
    using System;
    using System.Data.Entity;
    using System.Linq;
    using DDS_Restaurant_Solution.Data;
    using DDS_Restaurant_Solution.Models;

    public class DataContext : DbContext
    {
        public DataContext() : base("name = DDS_Restaurant_Solution.Properties.Settings.cnnStr")
        {
            Database.SetInitializer(new CreateDatabaseIfNotExists<DataContext>());
        }
        public DbSet<Cargo> Cargos { get; set; }
        public DbSet<Genero> Generos { get; set; }
        public DbSet<Sucursal> Sucursals { get; set; }
        public DbSet<TipoUsuario> TipoUsuarios { get; set; }
        public DbSet<Mesa> Mesas { get; set; }
        public DbSet<Pais> Pais { get; set; }
        public DbSet<Empleado> Empleados { get; set; }
        public DbSet<Departamento> Departamentos { get; set; }
        public DbSet<Municipio> Municipios { get; set; }
        public DbSet<Usuario> Usuarios { get; set; }
        public DbSet<Cliente> Clientes { get; set; }
        public DbSet<Menu> Menus { get; set; }
        public DbSet<DetalleMenu> DetalleMenus { get; set; }
        public DbSet<Producto> Productos { get; set; }
        public DbSet<TipoProducto> TipoProductos { get; set; }
        public DbSet<Pedido> Pedidos { get; set; }
        public DbSet<DetallePedido> DetallePedidos { get; set; }

        
    }

    //public class MyEntity
    //{
    //    public int Id { get; set; }
    //    public string Name { get; set; }
    //}
}

File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/DataContext.vb
Imports System
Imports System.Data.Entity
Imports System.Linq

Public Class DataContext
    Inherits DbContext

    Public Sub New()
        MyBase.New("name = DDS_Restaurant_VB.My.MySettings.cnnStr")
        Database.SetInitializer(New CreateDatabaseIfNotExists(Of DataContext)())
    End Sub
    Public Property Cargos As DbSet(Of Cargo)
    Public Property Generos As DbSet(Of Genero)
    Public Property Sucursals As DbSet(Of Sucursal)
    Public Property TipoUsuarios As DbSet(Of TipoUsuario)
    Public Property Mesas As DbSet(Of Mesa)
    Public Property Pais As DbSet(Of Pais)
    Public Property Empleados As DbSet(Of Empleado)
    Public Property Departamentos As DbSet(Of Departamento)
    Public Property Municipios As DbSet(Of Municipio)
    Public Property Usuarios As DbSet(Of Usuario)
    Public Property Clientes As DbSet(Of Cliente)
    Public Property Menus As DbSet(Of Menu)
    Public Property DetalleMenus As DbSet(Of DetalleMenu)
    Public Property Productos As DbSet(Of Producto)
    Public Property TipoProductos As DbSet(Of TipoProducto)
    Public Property Pedidos As DbSet(Of Pedido)
    Public Property DetallePedidos As DbSet(Of DetallePedido)
    Public Property Precios As DbSet(Of Precios)

End Class

'Public Class MyEntity
'    Public Property Id() As Int32
'    Public Property Name() As String
'End Class
